Author:         Ganesh Sittampalam <ganesh@earth.li>
Forwarded:      http://bugs.darcs.net/patch184
Bug:            http://bugs.darcs.net/issue1508
Description:    add cabal flag to control build of darcs library
Index: darcs-2.4.1/darcs.cabal
===================================================================
--- darcs-2.4.1.orig/darcs.cabal	2010-05-07 15:48:40.371169582 +1000
+++ darcs-2.4.1/darcs.cabal	2010-05-07 15:48:43.436170000 +1000
@@ -97,6 +97,10 @@
   description: Use GADT type witnesses.
   default:     False

+flag library
+  description: Build darcs library
+  default:     True
+
 flag color
   description: Use ansi color escapes.

@@ -155,6 +159,11 @@
 -- ----------------------------------------------------------------------

 Library
+  if flag(library)
+    buildable: True
+  else
+    buildable: False
+
   hs-source-dirs:   src
   include-dirs:     src
